manim/animation/creation.py

```diff
@@ -58,7 +58,7 @@ def construct(self):
 
 __all__ = [
     "ShowPartial",
-    "ShowCreation",
```

Comment:
`ShowCreation` and `Create` both need to be in `__all__` for the duration of the deprecation so that either can be used with `from manim import *`.

Reply:
Thank you for the instructions, @jsonvillanueva!